Obsidian/Utilities/Mojang/MinecraftAPI.cs

```diff
@@ -1,49 +1,43 @@
-﻿using System.Net.Http;
+using System.Net.Http;
+using System.Net.Http.Json;
 using System.Text;
-using System.Text.Json;
 using System.Web;
 
 namespace Obsidian.Utilities.Mojang;
 
 public class MinecraftAPI
 {
-    private static readonly HttpClient Http = Globals.HttpClient;
+    private static readonly HttpClient httpClient = new();
```

Comment:
Why is a new instance made inside `MinecraftAPI`?

Reply:
Missed that. Meant to use the Global HttpClient when adding the Global Json Options

Reply:
Fixed as of commit 63905f2